notebooks/8_combine_dataframes.ipynb

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
<p>Overall, I had a bit of a hard time following this lesson because there are so many columns, and so many DataFrames with similar names, plus some DataFrames get overwritten. It's information overload, and I don't know where to focus my attention.</p><p> </p><p>In order for a beginner to follow, I would recommend removing most (but not all) of the irrelevant columns (perhaps during the file reading process), just so they can focus their attention on the relevant columns. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Good point. I'm going to clean up the example data sets, in order to limit the number of columns the user gets confronted with.</p>